addons/website_forum/data/forum_data.xml

```diff
@@ -13,7 +13,7 @@
         </record>
     </data>
     <data>
-        <function model="ir.config_parameter" name="set_param" eval="('auth_signup.allow_uninvited', True)"/>
+        <function model="ir.config_parameter" name="set_param" eval="('auth_signup.allow_uninvited', 'b2c')"/>
```

Comment:
Shouldn't we rename allow_uninvited ? Either we keep as a boolean, either we rename it, but I don't like keeping an old attribute with a boolean-like name set to a selection.

Reply:
Good point ! I would vote for a renaming, since we don't want to store something else in an ICP than a possible value for the field (that is a selection actually)

Reply:
fixed